starknet_py/tests/e2e/docs/quickstart/test_synchronous_full_node_client.py

```diff
@@ -3,6 +3,7 @@
 
 def test_synchronous_full_node_client(
     client,
+    map_class_hash,  # pylint: disable=unused-argument
```

Comment:
Could you please explain why was this param added?

Reply:
This line was added to generate the block that we get later in the test. It is probably correlated with devnet update

Reply:
Ok, thanks for clarification.